packages/webview_flutter/ios/Classes/FLTWKNavigationDelegate.m

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
2019? lol

Reply:
In this diff it looks like it is a new file. If it is indeed created in 2018, please ignore my comment.

Reply:
done